Documentation/ansible-install.md

```diff
@@ -0,0 +1,59 @@
+# Installation using openshift-ansible
+
+Using ansible is the recommend installation method for installing Metering on to an Openshift cluster.
+
+Within the [openshift-ansible repo][openshift-ansible] is where the [openshift-metering playbook][metering-playbook] is located.
```

Comment:
either

`Within the [openshift-ansible repo][openshift-ansible] is the [openshift-metering playbook][metering-playbook].`

or 

`The [openshift-metering playbook][metering-playbook] is located within the [openshift-ansible repo][openshift-ansible].`

Reply:
Yeah wow the existing phrasing was pretty bad. Thanks.